ci/kokoro/install/build.sh

```diff
@@ -17,15 +17,71 @@
 set -eu
 
 echo "================================================================"
-echo "Change working directory to project root."
+echo "Change working directory to project root $(date)."
 cd "$(dirname "$0")/../../.."
+
+
+echo "================================================================"
+echo "Load Google Container Registry configuration parameters $(date)."
+
+if [[ -f "${KOKORO_GFILE_DIR:-}/gcr-configuration.sh" ]]; then
+  source "${KOKORO_GFILE_DIR:-}/gcr-configuration.sh"
+fi
+
 echo "================================================================"
+echo "Setup Google Container Registry access  $(date)."
```

Comment:
Nit: Usually not two spaces before `$(date)`.

Reply:
Done.